sources/swap/config.move

```diff
@@ -0,0 +1,138 @@
+module liquidswap::config {
+    use std::signer;
+
+    use liquidswap::curves;
+
+    friend liquidswap::liquidity_pool;
+
+    // Error codes.
+
+    /// When config doesn't exists.
+    const ERR_CONFIG_DOES_NOT_EXIST: u64 = 300;
+
+    /// When user is not admin
+    const ERR_NOT_ADMIN: u64 = 301;
+
+    /// When invalid fee amount
+    const ERR_INVALID_FEE: u64 = 302;
+
+    // Constants.
+
+    /// Minimum value of fee.
+    const MIN_FEE: u64 = 1;
+
+    /// Maximum value of fee.
+    const MAX_FEE: u64 = 35;
+
+    /// Minimum value of dao fee.
+    const MIN_DAO_FEE: u64 = 0;
+
+    /// Maximum value of dao fee.
+    const MAX_DAO_FEE: u64 = 100;
```

Comment:
Now you have duplicated set of constants? The same ones are present in `liquidity_pool.move`

Reply:
Add `is_valid_fee` and `is_valid_dao_fee` functions.